data/gaming.html

```diff
@@ -41,9 +41,9 @@ <h2><a
                                                 class="material-icons">help</span></a>&zwnj;Lutris&zwnj;</h2>
                                     <div class="card card-block">
                                         <div class="card-image text-md-center">
-                                            <a href="img/logos/lutris.png" data-toggle="lightbox"><img
+                                            <img
```

Comment:
Stylistically lets have the img and the first line together rather than a new line.

Reply:
Sorry didn't see these earlier.
formatted.